### ISSUE TITLE
Add basic contribution docs

### ISSUE THREAD

**Issue by fubhy** [bufbuild/protobuf-es#399]
It would be great to have some basic readme with installation & contribution instructions. 

I just spent a solid hour trying to figur eout why `make` wasn't happy with me until I realized that I had recently updated to bazel v6 and this repository requires bazel 5.4.0 :-P

**Comment by smaye81** [comment]
We actually have them [here](https://github.com/bufbuild/protobuf-es/blob/main/.github/CONTRIBUTING.md).  Were these not readily available when trying to contribute?

**Comment by fubhy** [comment]
Oh, interesting. I didn't check there. I was looking for a link to such a document in the root README. That's where I'm used to finding those. Maybe that's just me?

**Comment by fubhy** [comment]
Let me rephrase then. Can we link to those from the README at the root? :)

**Comment by timostamm** [comment]
The CONTRIBUTING.md is picked up by GitHub. If you open an issue or go to the PR tab, you get a notification.

We went with it in all repositories so we don't add even more noise to the top level directory, but that also means we rely on GH to surface it.

I think you might not have gotten it because you contributed before we added the file? 

<img width="337" alt="Screenshot 2023-02-24 at 14 10 15" src="https://user-images.githubusercontent.com/4289451/221186737-969c113f-a14f-4096-844c-baf3d6c95721.png">

<img width="720" alt="Screenshot 2023-02-24 at 14 10 52" src="https://user-images.githubusercontent.com/4289451/221186872-2d5d0f0a-4ab1-4cc0-b0b6-8c2b9e56fecf.png">

(Looks like I never contributed to rails!)

**Comment by fubhy** [comment]
Seems like it, yeah. Imho it's a bit of a chicken & egg situation though. When I go to the `Pull requests` tab, I likely already have sth. ready to be contributed and am about to open a PR with that. So at that point, I don't need installation documentation anymore. But before I can do that, I need to open an issue, discuss with the maintainers and, first and foremost, actually have a working environmnet locally. Imho, this is misplaced by github. I'd expect this to be displayed on the front page. Or maybe, the installation instructions belong into the README and CONTRIBUTION guidelines should only include guidelines for how good contribution etiquette. Not sure.

**Comment by smaye81** [comment]
Hey @fubhy.  I think we're going to close this issue as the docs we have now are probably sufficient for the time being.  We will raise this within the company as a possibility to update across our repos, but for now we don't want to make an exception here.  Thanks again.
